jetty-server/src/main/java/org/eclipse/jetty/server/handler/IPAccessHandler.java

```diff
@@ -109,6 +116,7 @@
     // true means nodefault match
     PathMap<IPAddressMap<Boolean>> _white = new PathMap<IPAddressMap<Boolean>>(true);
     PathMap<IPAddressMap<Boolean>> _black = new PathMap<IPAddressMap<Boolean>>(true);
+    Set<String> _connectorNames = Collections.emptySet();
```

Comment:
As this class is deprecated, I'd rather not add features to it and just update InetAccessHandler.   If you really want it updated... then do it exactly the same as InetAccessHandler with IncludeExcludeSet

Reply:
done

Reply:
i did fix the dump stuff but removed the other changes.